libgstc/libgstc.c

```diff
@@ -406,6 +406,31 @@ gstc_pipeline_inject_eos (GstClient * client, const char *pipeline_name)
   return ret;
 }
 
+GstcStatus
+gstc_pipeline_seek(const GstClient *client, const char *pipeline_name,
+    double rate, int format, int flags, int start_type, long start,
```

Comment:
is it `long` or `long long`?

Reply:
gst_element_seek uses gint64

Reply:
`long` may be 32 bits. `long long` is 64 bits.